purge-project/PurgeProject-18.3.sql

```diff
@@ -7,9 +7,10 @@
  *
  *	NOTE:  This script defaults to rolling back changes.
  *		To commit changes, set @commitChanges = 1.
+ *		To make changes WITHOUT A TRANSACTION, no possibility of rollback, and possible data corruption, set @commitChanges = 2
  */
 
-declare @commitChanges bit; --set @commitChanges = 1
+declare @commitChanges bit; --set @commitChanges = 1; --set @commitChanges = 2
```

Comment:
can a `bit` store a value of 2?

Reply:
My preference is for the same variable, since the modes are mutually-exclusive.